plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/constants/CodeTransformChatItems.kt

```diff
@@ -0,0 +1,251 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.codemodernizer.constants
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFile
+import software.aws.toolkits.jetbrains.services.codemodernizer.getModuleOrProjectNameForFile
+import software.aws.toolkits.jetbrains.services.codemodernizer.messages.Button
+import software.aws.toolkits.jetbrains.services.codemodernizer.messages.CodeTransformButtonId
+import software.aws.toolkits.jetbrains.services.codemodernizer.messages.CodeTransformChatMessageContent
+import software.aws.toolkits.jetbrains.services.codemodernizer.messages.CodeTransformChatMessageType
+import software.aws.toolkits.jetbrains.services.codemodernizer.messages.CodeTransformFormItemId
+import software.aws.toolkits.jetbrains.services.codemodernizer.messages.FormItem
+import software.aws.toolkits.jetbrains.services.codemodernizer.messages.FormItemOption
+import software.aws.toolkits.jetbrains.services.codemodernizer.model.CodeModernizerJobCompletedResult
+import software.aws.toolkits.jetbrains.services.codemodernizer.model.ValidationResult
+import software.aws.toolkits.jetbrains.services.cwc.clients.chat.model.FollowUpType
+import software.aws.toolkits.jetbrains.services.cwc.messages.FollowUp
+import software.aws.toolkits.resources.message
+import software.aws.toolkits.telemetry.CodeTransformPreValidationError
+
+const val DOC_URL = "https://docs.aws.amazon.com/amazonq/latest/aws-builder-use-ug/code-transformation.html"
```

Comment:
nit: Should we move this to `software.aws.toolkits.jetbrains.services.amazonq.Constants`?